### PR TITLE
Fixing non-working links at profile page

### LINKED ISSUE #5021: Fix hyperlinks not working on profile page
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step
* [ ]  🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
* [ ]  📝 **UPDATE**
Change `line 31` of (https://github.com/publiclab/plots2/blob/master/app/views/tag/show.html.erb)  to:
```
<% if !@related_tags.nil? %>
    <p> <%= render partial: 'tag/subscribe_button', locals:{tags: @related_tags } %></p>
  <% end %>
```
to ensure that an iteration does not occur on a nil value (that is, `tags.each` should only run when `tags` has a non-nil value).
**Current Behavior**
Clicking on the hyperlinks to a user's questions and activities on their profile page breaks the app and opens an error page.
Screenshot 2019-03-11 at 10 38 15 PM
Screenshot 2019-03-11 at 9 58 50 PM

**Expected Behavior**
Clicking on the hyperlinks on a user's profile page ("questions", for example) should take you to a page that shows the questions or activities the user has made.
Screenshot 2019-03-11 at 10 38 15 PM
Screenshot 2019-03-11 at 10 38 33 PM

* [ ]  💾 **Commit** your changes
* [ ]  🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.
2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

* [ ]  🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?
Leave a comment below!

### PR DESCRIPTION
**Pull request by shubhi2000**
Fixes #5021 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by shubhi2000** [comment]
@CleverFool77 yeah sure. Is it possible for me to make changes in the name now?

**Comment by IshaGupta18** [comment]
@gauravano I think this is good to be merged!

File: app/views/tag/show.html.erb [hunk 1]
```diff
@@ -28,7 +28,9 @@
 
   <%= render partial: "tag/show/questions" %>
   <%= render partial: "tag/show/related_tags" %>
-  <p> <%= render partial: 'tag/subscribe_button', locals:{tags: @related_tags } %></p>
+  <% if !@related_tags.nil? %>
```
Comment (madeofhuman) [inline app/views/tag/show.html.erb]: You've done a really good job here, @shubhi2000. One tip, though. With Ruby, you don't need to make an explicit check for `nil`, especially when, in your case, you're checking to see if a variable is not nil. To check if a variable is not nil, you can just do `if variable`, which will return true if it's not nil, or false if it's nil.

So, you can write your code like this:

```suggestion
  <% if @related_tags %>
```
It makes it a lot less complex to read.

Comment (shubhi2000) [reply app/views/tag/show.html.erb]: @madeofhuman thank you so much. I am completely new to Ruby, that's why.

Comment (madeofhuman) [reply app/views/tag/show.html.erb]: It's perfectly all right. 😄 I remember when someone gave me this same tip on my first Ruby project.

Comment (shubhi2000) [reply app/views/tag/show.html.erb]: Ohh, nice.
It is really helpful when someone gives us a tip, it benefits us in the long-term. Thanks!

**Comment by gauravano** [comment]
Merged :100: :tada: :balloon:

**Comment by shubhi2000** [comment]
Thank you soo much @gauravano, @CleverFool77, @madeofhuman , @IshaGupta18 for helping me out.

**Pull request merged**